now with completed assignment

bennettatoms opened the pull request:
{
completeness: 5,
comfortability: 4,
questions: n/a
}

bennettatoms committed:
now with completed assignment

drmikeh commented on seeds.sql:
```diff
@@ -0,0 +1,18 @@
+# seeds.sql that inserts some of your favorite TV shows into the shows table.
```
Should use -- instead of # for SQL comments.

drmikeh commented on create_schema.sql:
```diff
@@ -0,0 +1,6 @@
+CREATE TABLE new_shows (
```
table name should be "shows"

drmikeh commented:
I added 2 minor comments, but it looks good.
:+1:

bennettatoms commented:
Sorry, I created new_shows after the fact to troubleshoot the integer range problem and forgot to change it back. Corrected now, and sql comment format noted. Thanks!

> On Feb 3, 2015, at 7:36 PM, Michael Hopper notifications@github.com wrote:
> 
> I added 2 minor comments, but it looks good.
> 
> —
> Reply to this email directly or view it on GitHub https://github.com/bennettatoms/tv_shows_exercise/pull/1#issuecomment-72767610.

bennettatoms committed:
corrected table name and comment format
